src/address_form_config/address_book.js

```diff
@@ -29,6 +30,18 @@ export default [
         postcode: "#FormField_13"
       },
       stateMappings: defaultStateMappings
+    },
+    int: {
+      countryValue: {'Belgium': 'be', 'Canada': 'ca', 'Czechia': 'cz', 'Czech Republic': 'cz', 'Denmark': 'dk', 'France': 'fr', 'Germany': 'de', 'Ireland': 'ie', 'Netherlands': 'nl', 'Portugal': 'pt', 'Singapore': 'sg', 'Spain': 'es', 'Sweden': 'se', 'United Kingdom': 'gb', 'United States': 'us'},
```

Comment:
Maps the Select HTML value to the country code which the underlying widget expects.